docs/source/tutorials/configure_metrics.rst

```diff
@@ -63,34 +69,36 @@ Latency Metric
             {
                 "name": "latency",
                 "type": "latency",
-                "sub_type": "avg",
+                "sub_types": [
+                    {"name": "avg", "priority": 2, "goal": {"type": "percent-min-improvement", "value": 20}}
```

Comment:
Can we remove `priority` here in example? Or set it to 1? 2 is confusing me here.